test/functional/feature_fee_estimation.py

```diff
@@ -113,6 +116,14 @@ def check_estimates(node, fees_seen):
     check_raw_estimates(node, fees_seen)
     check_smart_estimates(node, fees_seen)
 
+
+def make_tx(wallet, utxo, feerate):
+    """Create a 1in-1out transaction with a specific input and feerate (sat/vb)."""
+    return wallet.create_self_transfer(
+        utxo_to_spend=utxo,
+        fee_rate=Decimal(feerate * 1000) / COIN,
+    )
```

Comment:
make_tx was transformed from send_tx. You should investigate what introduced send_tx. Maybe we skipped a backport, and should do that one first. Maybe we have no need for this function.